### PR TITLE
Updated abstract

### PR DESCRIPTION
**Pull request by paul2keller**
Edited the abstract to focus less on transparency (a divisive issue that needs to be addressed independently from the work of this group) and more in the purpose of the vocabulary.

### CODE REVIEW

File: OptOutVocab.adoc [hunk 1]
```diff
@@ -35,12 +33,14 @@ This document focuses on the **{vocabulary}** pieces of that statement by descri
 
 Asset:: A digital file or stream of data containing content and usually with associated metadata. 
 
-Rightsholder:: Person or organization that owns the legal rights to something. See https://en.wiktionary.org/wiki/rightsholder[Wiktionary].
+Rightsholder:: Person or organization that owns the legal rights to something. In this context preliminary holders of copyight and related rights.
```
Comment (llemeurfr) [inline OptOutVocab.adoc:36]: -> typo in "copyright"

File: OptOutVocab.adoc [hunk 2]
```diff
@@ -17,13 +17,11 @@ toc::[]
...
+This document provides a common vocabulary that seeks to standardize machine readable rights reservations and other rightholder opt-outs from using works protected by copyright and related rights in the context of AI/ML training and other forms of Text and Data Mininig (TDM). It has been developed to adress one of the issues identified in this April 2024 policy brief by https://openfuture.eu[Open Future]: https://openfuture.eu/wp-content/uploads/2024/05/240516considerations_of_opt-out_compliance_policies.pdf[Considerations for opt-out compliance policies by AI model developers]. The paper dentifies four different concepts that require consensus among stakeholders in order to ensure that opt-outs / rights reservation can be expressed in a way that is effective, scalable, and able to meet the needs of both rights holders and AI model developers. the four concepts are (in bold):
```
Comment (lrosenthol) [inline OptOutVocab.adoc:20]:
```suggestion
This document provides a common vocabulary that seeks to standardize machine readable rights reservations and other rightholder opt-outs from using works protected by copyright and related rights in the context of AI/ML training and other forms of Text and Data Mining (TDM). It has been developed to address one of the issues identified in this April 2024 policy brief by https://openfuture.eu[Open Future]: https://openfuture.eu/wp-content/uploads/2024/05/240516considerations_of_opt-out_compliance_policies.pdf[Considerations for opt-out compliance policies by AI model developers]. The paper identifies four different concepts that require consensus among stakeholders in order to ensure that opt-outs & rights reservation can be expressed in a way that is effective, scalable, and able to meet the needs of both rights holders and AI model developers. The four concepts are (in bold): 
```

File: OptOutVocab.adoc [hunk 3]
```diff
@@ -17,13 +17,11 @@ toc::[]
...
 
-This document focuses on the **{vocabulary}** pieces of that statement by describing a proposed Opt-Out Vocabulary that can be used to describe whether one or more assets may be used as part of a data mining or AI/ML training workflow. It is intended to be useful for both location-based as well as unit-based asset identifiers.
+This document focuses on the **{vocabulary}** piece of this statement by describing a proposed opt-out Vocabulary that can be used to describe whether one or more assets may be used as part of a data mining or AI/ML training workflow. The purpose of the vocabulary is to provide a stable set of categories that can be used by rightholders to specify the scope of opt-outs / rights reservations they wish to make. A stable, well defined set of rights reservations will ensure that opt-out reservation can be exchanged across value change and that different systems to communicate, process and store opt-oyt / rights reservations can be deigned in an interoperable way. The vocabulary is implementation agnostic and can be implemented in a variety of opt-out schemes including `location-based` and `unit-based` schemes.
```
Comment (lrosenthol) [inline OptOutVocab.adoc:24]:
```suggestion
This document focuses on the **{vocabulary}** piece of this statement by describing a proposed opt-out Vocabulary that can be used to describe whether one or more assets may be used as part of a data mining or AI/ML training workflow. The purpose of the vocabulary is to provide a stable set of categories that can be used by rightholders to specify the scope of opt-outs / rights reservations they wish to make. A stable, well defined set of rights reservations will ensure that opt-out reservation can be exchanged across value change and that different systems to communicate, process and store opt-out & rights reservations can be designed in an interoperable way. The vocabulary is implementation agnostic and can be implemented in a variety of opt-out schemes including `location-based` and `unit-based` schemes. 
```

File: OptOutVocab.adoc [hunk 4]
```diff
@@ -35,38 +33,38 @@ This document focuses on the **{vocabulary}** pieces of that statement by descri
...
 
 Unit-based identifiers:: A machine-readable piece of information that identifies a single asset (unit), regardless of where the asset is located, either through the embedding of that information directly into the asset or via a separate database or registry.
 
-== Use Cases
+AI/ML Training:: The pre-training, training and fine-tuning of AI Models before their release. 
+
+== Structure
+
+The vocabulary consists of the overarching Text and Data Mining (`TDM`) category and a number or more specific use cases that can be independently adressed. The overarching `TDM` category is derived from the defintion of Text and Data minining in https://eur-lex.europa.eu/eli/dir/2019/790/oj#art_2.tit_1[Article 2(2) of the European Union's Copyright in the Digital Single Market Dorective]. For the purposes of this vocabulary TDM is not understood to cover indexing of content for the purpose of online search. This use case is adressed via the exiting https://datatracker.ietf.org/doc/html/rfc9309[Robots Exclusion Protocol]. Currently the vocabulary contains two use cases that relate to the use of assets for AI/ML training purposes.
```
Comment (lrosenthol) [inline OptOutVocab.adoc:46]:
```suggestion
The vocabulary consists of the overarching Text and Data Mining (`TDM`) category and a number or more specific use cases that can be independently adressed. The overarching `TDM` category is derived from the definition of Text and Data mining in https://eur-lex.europa.eu/eli/dir/2019/790/oj#art_2.tit_1[Article 2(2) of the European Union's Copyright in the Digital Single Market Directive]. For the purposes of this vocabulary, TDM is not understood to cover indexing of content for the purpose of online search. This use case is addressed using a `location-based` schema such as the exiting https://datatracker.ietf.org/doc/html/rfc9309[Robots Exclusion Protocol]. 

Currently the proposed vocabulary contains two use cases that relate to the use of assets for AI/ML training purposes.
```

File: OptOutVocab.adoc [hunk 5]
```diff
@@ -35,38 +33,38 @@ This document focuses on the **{vocabulary}** pieces of that statement by descri
...
 
-The following categories are proposed for use in the Opt-Out Vocabulary, based on the set of use cases identified in the previous section:
+The following categories are proposed for use in the Opt-Out Vocabulary.
 
-Search and Discovery:: The act of indexing the content (and/or metadata) of assets for the purpose of retrieval.
+TDM:: Text and Data Mining. The act of using one or more assets in the context of any automated analytical technique aimed at analysing text and data in digital form in order to generate information which includes but is not limited to patterns, trends and correlations for purposes other than scientific reserach.
 
-TDM:: Text and Data Mining. The Copyright in the Digital Single Market (CDSM) Directive defines TDM as "any automated analytical technique aimed at analysing text and data in digital form in order to generate information which includes but is not limited to patterns, trends and correlations" (Article 2.2).
+AI Training:: The act of training AI/ML models that are NOT capable of generating text, images, and other synthetic content on one or more assets as input. examples are models than can be used for classification or object detection.
```
Comment (lrosenthol) [inline OptOutVocab.adoc:56]:
```suggestion
AI Training:: The act of training AI/ML models that are NOT capable of generating text, images, and other synthetic content on one or more assets as input. Examples are models than can be used for classification or object detection.
```

File: OptOutVocab.adoc [hunk 6]
```diff
@@ -35,38 +33,38 @@ This document focuses on the **{vocabulary}** pieces of that statement by descri
...
 
-AI Training:: The act of training an AI/ML (Artificial Intelligence/Machine Learning) model using one or more assets as input. This can include training for classification, object detection, as well as generative AI.
+Generative AI Training:: The act of training AI/ML models that are capable of generating text, images, and other synthetic content on one or more assets as input.
 
-Generative AI Training:: A form of `AI Training` where the AI/ML model being trained can generate new assets based on the training data.
-
-In addition to the pre-defined categories, it is also expected that some systems may extend this list with additional categories for their particular needs.
+This list of categories may be expanded with categories adressing additional uses cased should such use cases emerge. In addition to these categories definined in the vocabulary, it is also expected that some systems implementing this vocabulary may extend this list with additional categories for their particular needs.
```
Comment (lrosenthol) [inline OptOutVocab.adoc:60]:
```suggestion
This list of categories may be expanded with categories addressing additional uses cases should such use cases emerge. In addition to these categories defined in the vocabulary, it is also expected that some systems implementing this vocabulary may extend this list with additional categories for their particular needs. 
```

File: OptOutVocab.adoc [hunk 7]
```diff
@@ -35,38 +33,38 @@ This document focuses on the **{vocabulary}** pieces of that statement by descri
...
 
 === Relationship
 
-`Search and Discovery` is a category that is separate from the others, as it is not a form of TDM or AI Training. It is however reflected here as some opt-out systems (i.e., `robots.txt`) include this category.
-
-The `TDM` category is the overarching category that includes not only its own use cases, but also the various types of AI training, as they are considered to be forms of TDM.  As such, if a rightsholder opts out of TDM, they are opting out of those other categories as well.
+The `TDM` category is the overarching category that includes not only its own use cases, but also the various types of AI training, as they are considered to be forms of TDM.  As such, if a rightsholder opts out of / reserves the rights to `TDM`, they are opting out of those other categories as well. The other categories can be independtly adressed and affect only the specifi use case covered by the category.
```
Comment (lrosenthol) [inline OptOutVocab.adoc:64]:
```suggestion
The `TDM` category is the overarching category that includes not only its own use cases, but also the various types of AI training, as they are considered to be forms of TDM.  As such, if a rights-holder opts out of or reserves the rights to `TDM`, they are opting out of those other categories as well. The other categories can be independently addressed and affect only the specific use case covered by the category. 
```

**Review by lrosenthol — CHANGES_REQUESTED** [review]
a bunch of grammar and spelling changes

**Pull request merged**